### PR TITLE
feat: bump cloud-toolkit and improve doc

### PR DESCRIPTION
**Pull request by Scra3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,28 +20,56 @@ then go to the generated folder
...
+**[Read the developer guide](https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization)** to learn how to write your customizations.
+
+# Publish your customizations
+
+To publish your customizations, you need to build, package and publish your customizations.
+The build command will compile your customizations, the package command will create a package and the publish command will publish your package to your project.
 
-# Push your customizations to your project
+```bash
+forest-cloud build
```
Comment (ghusse) [inline README.md:42]: Should we indicate the version with `npx @forestadmin/cloud-toolkit`  here? Chances are that the package won't be globally installed.

File: README.md [hunk 2]
```diff
@@ -20,28 +20,56 @@ then go to the generated folder
...
 # Update typings
 
-`yarn update-typings` or `npm run update-typings`
+`forest-cloud update-typings`
+
+You can also use `yarn` or `npm` and run `forestadmin:update-typings` script.
 
 This command will update typings according to the structure of your database and your current code customizations.
-The typings are here to provide autocompletion in your IDE to help you write faster and less error prone code.
+The typings are here to provide autocompletion in your IDE to help you write faster and less error in your code.
```
Comment (ghusse) [inline README.md:56]:
```suggestion
The typings are here to provide autocompletion in your IDE to help you write faster and avoid errors in your code.
```

File: README.md [hunk 3]
```diff
@@ -20,28 +20,56 @@ then go to the generated folder
...
 and install the dependencies
 
-`yarn` or `npm install`
+`yarn install` or `npm install`
+
+then build the project
+
+`yarn build` or `npm run build`
+
+**You can found in your `package.json` some scripts to help you to develop your customizations.
```
Comment (ghusse) [inline README.md:29]:
```suggestion
**You can found in your `package.json` some scripts to help you to develop customizations.
```

File: README.md [hunk 4]
```diff
@@ -20,28 +20,56 @@ then go to the generated folder
...
+You can replace them with your own scripts if you want by using directly the `forest-cloud` command.**
```
Comment (ghusse) [inline README.md:30]:
```suggestion
You can replace them with your own scripts, by using directly the `forest-cloud` command.**
```

File: README.md [hunk 5]
```diff
@@ -20,28 +20,56 @@ then go to the generated folder
...
 
 # Write your customizations
 
-Read the developer guide: https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization
+**[Read the developer guide](https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization)** to learn how to write your customizations.
```
Comment (ghusse) [inline README.md:34]:
```suggestion
**[Read the developer guide](https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization)** to learn how to write customizations.
```

File: README.md (hunk 1 above)
Comment (Scra3) [reply README.md]: In the previous section I ask to the user to install the dependencies, it means we can use the version locally by adding npx in front of each commands

**Pull request merged**